elixir-sdk/lib/core.ex

```diff
@@ -0,0 +1,37 @@
+defmodule EppoSdk.Core do
```

Comment:
This module just defines the Rust interface -- it was confusing to me at first, but all these functions are "replaced" by Rust functions with the same names.